vagen/envs_remote/handler.py

```
@@ -0,0 +1,342 @@
+            HandlerResult with session_id
+            If seed provided: also includes obs, info, and images from reset
+
+        Raises:
+            RuntimeError: If max_sessions limit reached
+        """
+        # Check session limit
+        if self.max_sessions > 0 and len(self._sessions) >= self.max_sessions:
+            raise RuntimeError(
+                f"Max sessions limit reached ({self.max_sessions}). "
+                f"Please try again later or close existing sessions."
+            )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)